kubemarine/plugins/__init__.py

```diff
@@ -522,8 +523,33 @@ def apply_expect(cluster, config, plugin_name=None):
 # **** PYTHON ****
 
 def verify_python(cluster, step):
-    # TODO: verify fields types and contents
-    return
+    module_path, _ = utils.determine_resource_absolute_file(step['module'])
+    method_name = step['method']
+    method_arguments = step.get('arguments', {})
+
+    # Try to import the module
+    try:
+        spec = importlib.util.spec_from_file_location('module', module_path)
+        module = importlib.util.module_from_spec(spec)
+        spec.loader.exec_module(module)
+    except Exception as e:
+        raise ValueError(f"Could not import module {module_path}: {e}")
+
+    # Check if the method exists
+    if not hasattr(module, method_name):
+        raise ValueError(f"Module {module_path} does not have method {method_name}")
+
+    # Get the method object
+    method = getattr(module, method_name)
```

Comment:
There is a code duplication. Instead, you could add new method `get_python_method_args() -> Tuple[callable, dict]` that will 1) import module and perform all validations; 2) return method and args. Use the method in both verify_python and apply_python.